app/src/main/java/com/example/yard/fragments/BioGridFragment.java

```diff
@@ -0,0 +1,44 @@
+package com.example.yard.fragments;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.yard.R;
+import com.example.yard.adapter.PostCreation;
+import com.example.yard.adapter.PostsAdapter;
+import com.example.yard.helperclass.PostQueryHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BioGridFragment extends Fragment {
+  protected RecyclerView rvPosts;
```

Comment:
Use correct modifiers. If you're not creating subclass for BioGridFragment, there's no need to declare them as protected.